### PR TITLE
Allow import for sql_quoting

### PR DESCRIPTION
**Pull request by jhinghaus**
We need to import DocumentTemplate.DT_Var to use the better sql_quoting.

### CODE REVIEW

File: Products/ZPerFactMods/allowScriptModules.py [hunk 1]
```diff
@@ -75,6 +75,8 @@
 # This is on by default in Zope2, but not in Zope4
 allow_module("Products.PythonScripts.standard")
 
+allow_module("DocumentTemplate.DT_Var")
```
Comment (viktordick) [inline Products/ZPerFactMods/allowScriptModules.py:79]: Maybe add a comment, but otherwise LGTM.

**Review by viktordick — APPROVED** [review]
Please add a comment and feel free to merge and increase and release the version (I guess to 3.15.0 if we want to continue to have versions between the different related packages the same, although we might want to abandon that practice).

**Comment by viktordick** [comment]
@jhinghaus sorry, I meant 3.14.0, not 3.15.0

**Review by jan-jockusch — COMMENTED** [review]
LGTM (after the fact)!

**Pull request merged**